router/initoutputs_test.go

```diff
@@ -153,8 +153,8 @@ func TestBuildAndInitOtpt(t *testing.T) {
 				"User":       "admin",
 				"Password":   "admin",
 				"ProjectKey": "PK",
-				"Issuetype":  IssueTypeDefault,
-				"Priority":   PriorityDefault,
+				"Issuetype":  "",
```

Comment:
isn't it a default behavior? Probably `Issuetype` and `Priority` could be just omitted

Reply:
Done